baseline/client/pay-info-page/index.js

```diff
@@ -24,15 +24,14 @@ getAuth(
             const targetId = parseTargetId(window.location.search) ?? idToken.sub;
             const client = new ApiClient(session);
             // determine what kind of access (admin/self/invalid)
-            if (hasPreferredRole(idToken, "pvs-dev-study-admin")) {
-                const payboard = new Payboard(payboardDiv, errorDiv, new ApiClient(session), targetId, true);
-                payboard.refresh();
-            } else if (targetId = idToken.sub) {
-                const payboard = new Payboard(payboardDiv, errorDiv, new ApiClient(session), targetId, false);
-                payboard.refresh();
-            } else {
-                throw new Error("can't access user");
-            }
+            const payboard = new Payboard(
+                payboardDiv,
+                errorDiv,
+                new ApiClient(session),
+                targetId,
+                hasPreferredRole(idToken, "pvs-dev-study-admin"),
```

Comment:
Sorry - I just realized that while I mentioned this string ('pvs-dev-study-admin') would change, I didn't say what should be done about that. It needs to go into common/aws-settings.json.tmpl , and then you can import aws-settings.json and access it that way.

Reply:
Changed to read from `aws-settings.json` in b597423aaa7718e8c371ef75d62fafe2969f83cd!